enterprise/internal/campaigns/webhooks.go

```diff
@@ -809,21 +809,28 @@ func (h *BitbucketServerWebhook) ServeHTTP(w http.ResponseWriter, r *http.Reques
 		return
 	}
 
-	pr, ev := h.convertEvent(e)
-	if pr == (PR{}) || ev == nil {
-		log15.Debug("Dropping Bitbucket Server webhook event", "type", fmt.Sprintf("%T", e))
-		respond(w, http.StatusOK, nil) // Nothing to do
-		return
-	}
-
 	externalServiceID, err := extractExternalServiceID(extSvc)
 	if err != nil {
 		respond(w, http.StatusInternalServerError, err)
 		return
 	}
 
-	if err := h.upsertChangesetEvent(r.Context(), externalServiceID, pr, ev); err != nil {
-		respond(w, http.StatusInternalServerError, err)
+	prs, ev := h.convertEvent(e)
+
+	m := new(multierror.Error)
+	for _, pr := range prs {
+		if pr == (PR{}) {
```

Comment:
Redundant parentheses?
```suggestion
		if pr == PR{} {
```

Reply:
Nope, they're required or it's a syntax error:

`Type PR is not an expression`

Reply:
I see, interesting! thanks